### ISSUE TITLE
Suggestion to rename `speech_detection` to `speech_recognition`

### ISSUE THREAD

**Issue by alexge233** [rapp-project/rapp-platform#234]
I would suggest we rename `speech_detection_google` and `speech_detection_sphinx` to `speech_recognition_X`.

The term `speech_detection` implies we're detecting speech, as in *diarization* which is a feature I would like to implement either locally in the API, or as a Service. It's purpose is to detect audio speech within audio files, whereas the Google and Sphinx4 API are recognizing speech as text.

**Comment by etsardou** [comment]
Indeed, you are right. If you plan to implement this in the near future, instead of ```speech detection``` use the more appropriate term ```voice activity detection```.

We will plan this change for the 0.7 release.

**Comment by alexge233** [comment]
How about `diarization` which is the keyword normally used?
I've found: [shout](http://shout-toolkit.sourceforge.net/download.html) which is a continuous diarization written in C++ as somebody's PhD research.

**Comment by etsardou** [comment]
Sure, as you prefer.
